drivers/clk/starfive/clk-starfive-jh7100-audio.c

```diff
@@ -112,7 +131,18 @@ static int jh7100_audclk_probe(struct platform_device *pdev)
 	if (IS_ERR(priv->base))
 		return PTR_ERR(priv->base);
 
-	for (idx = 0; idx < JH7100_AUDCLK_END; idx++) {
+	priv->pll[0] = clk_hw_register_fixed_rate(priv->dev, "i2sadc_bclk_iopad", NULL, 0, 12288000);
+	priv->pll[1] = clk_hw_register_fixed_rate(priv->dev, "i2sadc_lrclk_iopad", NULL, 0, 12288000);
+	priv->pll[2] = clk_hw_register_fixed_rate(priv->dev, "i2sdac_bclk_iopad", NULL, 0, 12288000);
+	priv->pll[3] = clk_hw_register_fixed_rate(priv->dev, "i2sdac_lrclk_iopad", NULL, 0, 12288000);
```

Comment:
Must these iopad clocks always be 12.288kHz? If not their frequency should probably be specified in the device tree.